search_service/config.py

```diff
@@ -26,17 +26,6 @@ class Config:
     # LOG_CONFIG_FILE = 'search_service/logging.conf'
     LOG_CONFIG_FILE = None
 
-    # Used to differentiate tables with other entities in Atlas. For more details:
```

Comment:
I'd still like to keep the config here and adjust the calling code in atlas_proxy.py to use the values from config instead.

Reply:
we discussed it that it might not make sense as this code will become more and more tied to amundsenatlastypes repo